doc/api/string_decoder.md

```diff
@@ -63,8 +63,8 @@ Creates a new `StringDecoder` instance.
 added: v0.9.3
 -->
 
-* `buffer` {Buffer|TypedArray|DataView} A `Buffer`, or `TypedArray`, or
-  `DataView` containing the bytes to decode.
+* `buffer` {string|Buffer|TypedArray|DataView} A `string`, or `Buffer`, or
```

Comment:
I think the descriptions here should be amended a bit to get rid of the excessive "or" usage. How about:

```
A `string`, `Buffer`, `TypedArray`, or `DataView` containing ...
```

Reply:
That's a great comment! I also wanted to briefly explain the types.